client/state/action-types.ts

```diff
@@ -857,6 +857,7 @@ export const SIGNUP_STEPS_SOCIAL_PROFILES_UPDATE = 'SIGNUP_STEPS_SOCIAL_PROFILES
 export const SIGNUP_STEPS_SOCIAL_PROFILES_RESET = 'SIGNUP_STEPS_SOCIAL_PROFILES_RESET';
 export const SIGNUP_STEPS_WEBSITE_CONTENT_REMOVE_LOGO_URL =
 	'SIGNUP_STEPS_WEBSITE_CONTENT_REMOVE_LOGO_URL';
+export const BBE_UPDATE_GENERIC_FEEDBACK = 'BBE_UPDATE_GENERIC_FEEDBACK';
```

Comment:
Just curious - why not follow the existing convention on action names (`SIGNUP_STEPS_WEBSITE_CONTENT...`)?

Reply:
Ah yeah good idea. I guess I was just caught up in the rebrand 🍭